vermouth/graph_utils.py

```diff
@@ -198,23 +241,20 @@ def initialize(self):
 
     def semantic_feasibility(self, node1, node2):
         # TODO: implement (partial) wildcards
-        elem1 = self.G1.node[node1].get('element', first_alpha(self.G1.node[node1]['atomname']))
-        elem2 = self.G2.node[node2].get('element', first_alpha(self.G2.node[node2]['atomname']))
+        elem1 = self.G1.node[node1]['element']
+        elem2 = self.G2.node[node2]['element']
         return elem1 == elem2
 
 
 def blockmodel(G, partitions, **attrs):
     """
     Analogous to networkx.blockmodel, but can deal with incomplete partitions,
-    assigns ``attrs`` to nodes, and calculates the new ``position`` attribute as
-    center of geometry.
+    and assigns ``attrs`` to nodes.
 
     Parameters
     ----------
     G: networkx.Graph
         The graph to partition
-    partitions: iterable of iterables
```

Comment:
Shouldn’t that remains?

Reply:
Yes it should. I'm not sure why it disappeared...